docs/content_management/field_types/field_type_reference/binaryfilefield.md

```diff
@@ -26,33 +26,34 @@ Note that both `BinaryFile` and `Media` Value and Type inherit from the `BinaryB
 |`mimeType`|string|The file's MIME type.|application/pdf|
 |`uri`|string|The binary file's `content/download` URI. If the URI doesn't include a host or protocol, it applies to the request domain.|/content/download/210/2707|
 |`downloadCount`|integer|Number of times the file was downloaded|0|
```

Comment:
Maybe `inputUri` could replace `path` here. But as `inputUri` is mandatory in the construction hash, and wont change afterward, I don't see the interest. I would keep only what is useful when consulting the value of an existing field.
```suggestion
|`downloadCount`|integer|Number of times the file was downloaded|0|
|`inputUri`|string|Path to a local file when creating a field value, `null` when reading a field value|`path/to/document.pdf`|
```